ci-operator/step-registry/baremetalds/e2e/ovn/ipsec-pre414/baremetalds-e2e-ovn-ipsec-pre414-workflow.metadata.json

```diff
@@ -8,4 +8,4 @@
 			"sdn-reviewers"
 		]
 	}
-}
```

Comment:
this commit could be removed if you modify the original commit which added these files to have the right
line endings. just a nit